jwst/pipeline/level2association.py

```diff
@@ -0,0 +1,100 @@
+"""Open a Level2 association given various inputs"""
```

Comment:
Fascinating. Is "@#$@#$@" a new kind of command that automatically gets fed to Travis CI?